src/components/pages/OfflineVerification/PageVerificationInfo.tsx

```diff
@@ -163,6 +163,7 @@ const PageVerificationInfo = ({
         doVerifyMerkleProof,
         null
       );
+      console.log({ verificationStatus, details });
```

Comment:
Accidentally committed.

Reply:
thanks, will resolve in upcoming commit. This should be caught by a linter in the future.

Reply:
made a note to add some linting. will handle it in this PR or in a small followup